packages/zoe/src/contracts/helpers/zoeHelpers.js

```diff
@@ -5,6 +5,9 @@ import { sameStructure } from '@agoric/same-structure';
 export const defaultRejectMsg = `The offer was invalid. Please check your refund.`;
 export const defaultAcceptanceMsg = `The offer has been accepted. Once the contract has been completed, please check your payout`;
 
+// Must not escape
+const OfferResultPumpkin = harden({});
```

Comment:
I really like the simplifications in `makeInvitePair`, but I'm not sure I like the `OfferResultPumpkin` as shared data. Could we put it inside the `makeInvitePair` function?